parts/common/components.json

```diff
@@ -42,6 +47,11 @@
           "renovateTag": "registry=https://mcr.microsoft.com, name=windows/nanoserver",
           "latestVersion": "ltsc2022",
           "windowsSkuMatch": "23H2*"
+        },
+        {
+          "renovateTag": "registry=https://mcr.microsoft.com, name=windows/nanoserver",
+          "latestVersion": "ltsc2022",
```

Comment:
Change `latestVersion` to [ltsc2025 ](https://hub.docker.com/r/microsoft/windows-servercore)?

Reply:
It may need to add both ltsc2022 or ltsc2025 if WS2025 host supports both 2022 containers and 2025 containers instead of updating this line.